launcher/launcher.go

```diff
@@ -242,7 +241,7 @@ func (l *launcher) reconfigureWait(timeout time.Duration) (*config.C, error) {
 				err := l.validator.Validate(update)
 				if err != nil {
 					l.log.Errorf("Config update validation failed: %v", err)
-					heatlhErr := &cb_errors.BeaterUnhealthyError{}
+					heatlhErr := &BeaterUnhealthyError{}
```

Comment:
It's before your change but can you plz fix the typo?
```suggestion
					healthErr := &BeaterUnhealthyError{}
```